home/tests/test_assessment_import_export.py

```diff
@@ -530,21 +531,37 @@ def test_invalid_high_inflecton_csv_format(self, csv_impexp: ImportExport) -> No
         )
         assert e.value.row_num == 2
 
-    def test_extra_columns(self, csv_impexp: ImportExport) -> None:
+    def test_extra_columns_csv(self, csv_impexp: ImportExport) -> None:
         """
         Importing a csv with an extra comma so there are more
         column values than headers should return an intuitive error message
 
         (This uses extra_columns.csv)
         """
 
-        with pytest.raises(ImportAssessmentException) as e:
+        with pytest.raises(ImportException) as e:
             csv_impexp.import_content_file("assessment_results.csv", purge=False)
             csv_impexp.import_file("extra_columns.csv")
-        assert (
-            e.value.message == "Invalid format. Please check that all row values "
-            "have headers."
-        )
+        assert e.value.message == [
+            "Invalid format. Please check that all row values have headers."
+        ]
+
+    def test_extra_columns_xlsx(
+        self, xlsx_impexp: ImportExport, csv_impexp: ImportExport
+    ) -> None:
+        """
+        Importing a xlsx with an extra comma so there are more
+        column values than headers should return an intuitive error message
+
+        (This uses extra_columns.xlsx)
+        """
+
+        with pytest.raises(ImportException) as e:
+            csv_impexp.import_content_file("assessment_results.csv", purge=False)
+            xlsx_impexp.import_file("extra_columns.xlsx")
+        assert e.value.message == [
+            "Invalid format. Please check that all row values " "have headers."
```

Comment:
```suggestion
            "Invalid format. Please check that all row values have headers."
```

I'm guessing the formatter combined these lines or something.